web2.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <title>Joe Johnson portfolio</title>
+    <link rel="stylesheet" type="text/css" href="css/style2.css" />
+</head>
+
+<header> ABOUT ME PAGE | <a href="web.html">portfolio</a> </header>
```

Comment:
this got me confused haha! The Portfolio link redirects to the about me page and vice versa. It should be the opposite.